contrib/docker/testnet.sh

```diff
@@ -74,6 +72,28 @@ set_repo()
   fi
 }
 
+set_bins()
+{
+  if [[ -z $KOVRI_USE_REPO_BINS ]]; then
+    read -r -p "Use binaries from repo ? [N/y] " REPLY
+    case $REPLY in
+      [yY])
+        KOVRI_USE_REPO_BINS=true
+        ;;
+      *)
+        KOVRI_USE_REPO_BINS=false
+      ;;
+     esac
+  fi
+
+  mount_repo_bins=""
+  if [[ "$KOVRI_USE_REPO_BINS" = true ]]; then
+    mount_repo_bins="-v ${KOVRI_REPO}/build/kovri:/usr/bin/kovri -v ${KOVRI_REPO}/build/kovri-util:/usr/bin/kovri-util"
+
```

Comment:
If we're going to start bool'ing, how about we create bool vars for building? This [bool'ing] can then be applied elsewhere so we never have to enter keyboard input once they are all set.

What do you think? If we apply elsewhere, I think it should be in a separate commit.